### ISSUE TITLE
Wrong warnings from Resizetizer More than one 'MauiSplashScreen' is defined

### ISSUE THREAD

**Issue by JoacimWall** [dotnet/maui#26237]
### Description

I have different setting for Android and IOS regarding the ForegroundScale for the appicon

 
        <MauiIcon Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'android'" Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="1.09" BaseSize="456,456" Color="#F5F0EB" />
        <MauiIcon Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'" Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="0.65" BaseSize="456,456" Color="#F5F0EB" /> 
        
        <MauiSplashScreen Include="Resources\Splash\splash.svg" BaseSize="400,146" Color="#F5F0EB" />

### Steps to Reproduce

1. create a new MAUI app
2. create two lines for the Appicon in the project file
 
        <MauiIcon Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'android'" Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="1.09" BaseSize="456,456" Color="#F5F0EB" />
        <MauiIcon Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'" Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="0.65" BaseSize="456,456" Color="#F5F0EB" /> 
        
        <MauiSplashScreen Include="Resources\Splash\splash.svg" BaseSize="400,146" Color="#F5F0EB" />
3. build and you will et worrning

### Version with bug

9.0.10 SR1

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android

### Did you find any workaround?

no

### Relevant log output

```shell

```

**Comment by StephaneDelcroix** [comment]
what's the warning you get ?

**Comment by JoacimWall** [comment]
More than one 'MauiSplashScreen' is defined

![Image](https://github.com/user-attachments/assets/b7274724-06f9-461a-bad5-22ef7b5d1f8c)

**Comment by mattleibow** [comment]
I tested this and cannot reproduce. Are you able to share a sample where this happens?

**Comment by JoacimWall** [comment]
Hi 
Wh have a senario where we want to remove the image and only use empty page no image. 
<MauiSplashScreen BaseSize="400,146" Color="#F5F0EB" />  //remove this Include="Resources\Splash\splash.svg"
this result in load field for the project file in Rider. And you are not able to export an empty svg from Figma. 

Se we changed the splash.svg build action from MAUISpashScreeen to None and this resulted in that (rider added this to the solution file. two new rows I the projectfile that we did not see) 

        <None Remove="Resources\Splash\splash.svg" />
        <MauiSplashScreen Include="Resources\Splash\splash.svg" />



I apologize for this, the extra work this entailed. We can close this ticket
